provider/resource_opensearch_monitor_test.go

```diff
@@ -71,52 +94,99 @@ func testCheckOpensearchMonitorDestroy(s *terraform.State) error {
 	return nil
 }
 
+var testAccOpensearchOpenDistroMonitorJSON = `
```

Comment:
had to duplicate this cuz the linting was complaining if I did it the smart way by embedding the variable.